idaes/power_generation/carbon_capture/compression_system/compressor.py

```diff
@@ -113,6 +117,11 @@ def polytropic_efficiency_eqn(b, t):
 class CompressionStageData(CompressorData):
     # Pressure changer with isentropic compressor options
     CONFIG = CompressorData.CONFIG()
+    CONFIG.declare("inlet_phase", ConfigValue(
+        default='Vap',
```

Comment:
This does not really resolve the issue at hand, and in fact could make it worse. All of the calculations are based on the assumption of a single vapour phase, so we probably need to check this and raise an Exception if it is not the case. I.e. add the following to `build`:

```
if self.config.property_package.phase_list != ["Vap"]:
    raise ConfigurationError(
        "{} SCO2 compressor provided with invalid property package. "
        "The SCO2 compressor only support a single vapor phase."
        .format(self.name))
```

As it stands, this would allow a user to provide a property package with an arbitrary number of phases, but would ignore anything other than the indicated phase ("Liq" or "Vap").

Reply:
Hi @andrewlee94 : I upload the revised version. Could you take a look at it again? Thanks!